src/components/data/data-utils.js

```diff
@@ -259,7 +259,7 @@ export const EntityUploader = ({ onSuccess, onDismiss, namespace, name, entityTy
           notifyDataImportProgress(jobId)
         }
       }
-      
+
```

Comment:
I like this change, but not in a file that would otherwise be untouched.

Reply:
Should I revert it then? NP if so, just wanted to check

Reply:
Yes. Sorry about not being clear.